workflow_scheduler.go

```diff
@@ -26,32 +39,83 @@ type ScheduleDescription struct {
 	WinEventSubscribeQueries [][]string `json:"win_event_subscribe_queries"`
 }
 
-/** Class that represents a query for subscribing to a windows event. */
-type WinEventSubscribeQuery struct {
-	channel string
-	query string
+/** Class that holds information used in the subscription callback function. */
+type CallbackContext struct {
+	workflow workflows.InterfaceWorkflow
+	workflowContext *nodes.WorkflowContext
+}
+
+func (workflowScheduler *WorkflowScheduler) SubscriptionCallback(Action wevtapi.EVT_SUBSCRIBE_NOTIFY_ACTION, UserContext win32.PVOID, Event wevtapi.EVT_HANDLE) uintptr {
+	var CStringGuid *C.char = (*C.char)(unsafe.Pointer(UserContext))
+	var guid string = C.GoStringN(CStringGuid, 36)
```

Comment:
Remove magic number

Reply:
Removed by using C.GoString instead of GoStringN